lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1346,7 +1346,7 @@ def test_view_certificate_link_hidden(self):
         self.assertContains(resp, u"Download Your Certificate")
 
     @ddt.data(
-        *itertools.product(((49, 4, True), (49, 4, False)), (True, False))
+        *itertools.product(((38, 4, True), (38, 4, False)), (True, False))
     )
     @ddt.unpack
     def test_query_counts(self, (sql_calls, mongo_calls, self_paced), self_paced_enabled):
```

Comment:
I don't think I knew you could use tuple-unpacking in a function/method signature.  Interesting.